Codeinwp/otter-blocks
# Add Default Value to Form Fields

Closes #1908


### Summary


Add default value component to Form fields.

🔧 New UI for Radio, Select, Checkbox in the Options to support selecting the default option value.

### Screenshots 

https://github.com/Codeinwp/otter-blocks/assets/17597852/864b1620-7098-4402-985c-22627aae154d

----

### Test instructions


Check the default value for Text type input (email, text, number, etc.) and Choice type (Radio, Select, Checkbox).



---- 

### Checklist before the final review
